src/FakeItEasy/Expressions/ArgumentConstraints/EqualityArgumentConstraint.cs

```diff
@@ -30,7 +30,7 @@ public override string ToString()
             var stringValue = this.ExpectedValue as string;
             if (stringValue != null)
             {
-                return "\"{0}\"".FormatInvariant(stringValue);
+                return '"' + stringValue + '"';

```

Comment:
I don't feel strongly about it, it was just an idea... the new style looks good to me 👍